pkg/reconciler/ducktype/ducktype.go

```diff
@@ -20,17 +20,10 @@ import (
 	"context"
 
 	corev1 "k8s.io/api/core/v1"
-	apierrs "k8s.io/apimachinery/pkg/api/errors"
-	corev1listers "k8s.io/client-go/listers/core/v1"
-
 	samplesv1alpha1 "knative.dev/discovery/pkg/apis/discovery/v1alpha1"
```

Comment:
shouldn't this be `discoverv1alpha1` or something similar?

Reply:
lol yes yes